LootMogul.md

RajNextgentm opened the pull request:
### Project Abstract

> Please provide a brief description of your project here summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.

### For which grant [level](https://github.com/w3f/Grants-Program#level_slider-levels) are you applying?

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $50,000, 3 approvals
- [x] **Level 3**:  Unlimited, 5 approvals (for > $100k Web3 Foundation Council approval)

### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
- [x] I have read and understood the [FAQs](https://github.com/w3f/Grants-Program/blob/master/docs/faq.md), [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/grant_guidelines_per_category.md) and [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/announcement-guidelines.md).
- [ ] A BTC, Ethereum (USDT/USDC/DAI) or Polkadot/Kusama (aUSD) address for the payment of the milestones is provided inside the application.
- [x] I have read and acknowledge the [terms and conditions](https://github.com/w3f/Grants-Program/blob/master/docs/T&Cs.md).
- [ ] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [ ] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process).
- [ ] I prefer the discussion of this application to be in a private Element/Matrix channel. My username is: ``@_______``

### How Did You Hear About our grants program?

- [x] Social Media
- [ ] Hackathon
- [ ] Personal Recommendation
- [ ] Substrate Builders Program
- [x] Investor/VC
- [x] Online Search
- [ ] Other: _______

Noc2 reviewed (changes requested):
Thanks for the application. But you are currently trying to merge two files. Could you remove the first one? Also deliverables 0a-0d of the [template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) are usually mandatory for all milestones, and deliverable 0e at least for the last one. Also given this size of the grant, it would be nice to have more technical details. For example, why does your tech stack mention Polygon Matic and nothing related to polkadot (pallets or ink! Smart contract)?

RajNextgentm reviewed (commented):
Updated milestone details as requested.

RajNextgentm commented:
> Thanks for the application. But you are currently trying to merge two files. Could you remove the first one? Also deliverables 0a-0d of the [template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) are usually mandatory for all milestones, and deliverable 0e at least for the last one. Also given this size of the grant, it would be nice to have more technical details. For example, why does your tech stack mention Polygon Matic and nothing related to polkadot (pallets or ink! Smart contract)?


Hi @Noc2 , 

Thank for your time reviewing our aplication. 

We have updated milestone details as requested. also removed an additional file which we were trying to merge.

Also, answering to your question regarding tech stacks, currently our platform is built on polygon network. Moving ahead we have milestones to migrate on polakadot ecosystem in next 6 months.
You can take a look at our platform visiting https://lootmogul.com/.
We are working in the space of play 2 earn, NFT and metaverse.

Noc2 reviewed (changes requested):
Thanks for the reply. Could you update the application accordingly?

RajNextgentm commented:
> Thanks for the reply. Could you update the application accordingly?

Application has been updated, kindly take a look and let us know if you need more information on this.

Noc2 reviewed (changes requested):
Thanks for the update. I have a few more questions: 

- Could you specify what you actually need to develop as part of the deliveries? For example is the stablecoin integration just in your front-end or do you develop some kind of pallet for it? 
- Could you remove the parts where you just plan on using existing code (like frontier)? We only support the development of open source software and don’t support the deployment of existing software. 
- What does KOL mean? 
- What exact parts will be integrated in Rust, Solidity, etc.? 
- Are you going to use ink!? 
- If you have existing open source software that you want to bring to our ecosystem as part of this grant, feel free to link to it as part of the deliveries. 
- In general, please add more details. For example what does “Game development and Guild integration” actually mean.

RajNextgentm commented:
> Thanks for the update. I have a few more questions:
> 
> * Could you specify what you actually need to develop as part of the deliveries? For example is the stablecoin integration just in your front-end or do you develop some kind of pallet for it?
> * Could you remove the parts where you just plan on using existing code (like frontier)? We only support the development of open source software and don’t support the deployment of existing software.
> * What does KOL mean?
> * What exact parts will be integrated in Rust, Solidity, etc.?
> * Are you going to use ink!?
> * If you have existing open source software that you want to bring to our ecosystem as part of this grant, feel free to link to it as part of the deliveries.
> * In general, please add more details. For example what does “Game development and Guild integration” actually mean.

Hi @Noc2 
Thanks for taking time and reviewing our proposal 

Below are the answers to your questions 

**Could you specify what you actually need to develop as part of the deliveries? For example is the stablecoin integration just in your front-end or do you develop some kind of pallet for it?**

Yes, it is just for frontend part as Web3, where we want user can purchase our NFTs and contest transactions using aUSD stable coin

**Could you remove the parts where you just plan on using existing code (like frontier)? We only support the development of open source software and don’t support the deployment of existing software.**

Sure, we will remove this part if it's not supported. We were referring to this article where we can write smart-contract using EVM pallet 
https://wiki.polkadot.network/docs/build-smart-contracts#substrate-contracts

**What does KOL mean?**

It’s a word we use for Key Opinion Leader (Influencers), We have 180 influencers onboard on our platform. 

**What exact parts will be integrated in Rust, Solidity, etc.?**

We will be using RUST for INK based smart contracts. Solidity is what we are currently using. We are building our platform on the polkadot ecosystem.

**Are you going to use ink!?**

Yes, we will be using ink! which is a domain specific language for writing smart contracts in Rust


**If you have existing open source software that you want to bring to our ecosystem as part of this grant, feel free to link to it as part of the deliveries.**

We don’t have an open source platform at the moment, however, we would like to bring our NFT market places, and in-game transactions to the polkadot ecosystem. We have built Lootmogul as play to earn gaming applications where we are building NFTs and Metaverse lands.

**In general, please add more details. For example what does “Game development and Guild integration” actually mean.**

We will be building and bringing more games for end users on our application where user can interact with, we will be also brining a lot of utilities in forms of metaverse lands, NFts

Noc2 reviewed (changes requested):
Thanks for the quick reply. I will mark the application as ready for review at this stage. But for me personally it’s still too unspecific given the price. For example, I don’t know what you will actually deliver as part of “Game development and Guild integration”.

takahser reviewed (changes requested):
@RajNextgentm thanks for the submission of this grant application.

Currently, it's difficult for me to understand, what you're planning to build. Could you be more specific and add more details?

Some examples:

> GitHub/Lab Polkadot dev branch & configure supporting APIs

What does that exactly involve? What kind of APIs are you planning to support here and how will they integrate, and what will they integrate with?

> Smart Contract libraries for NFT Marketplace

The lib will be traded as an NFT, or what's the connection here? What's the goal of these smart contract libraries? Will they allow somebody to write smart contracts, or are these smart contracts themselves?

> Ability to mint meta NFTs using INK for writing smart-contracts. we

Is there a reason you're not reusing existing NFT pallets, such as RMRK or Uniques? 

> Web 3 integration for the polkadot-js wallet.

What do you mean by that?

These are just some examples, so feel free to add more details in general.

cruikshankss commented:
Thank you for your application. I like the vision & business model. But I currently have low confidence in your technical expertise and ability to build this. If I were to approve this, I'd probably need an extensively detailed technical proposal. The ideal web 3.0 technical stack, especially in this ecosystem, is currently highly complicated and there are many ideals to fulfill. Without your express explanation of the technical complexity and technical ideals, unfortunately I have doubts in your ability to build your vision. 

I thank you for applying to w3f grants and I hope you can demonstrate the technical expertise probably needed for success, and then I would be happy to approve your application.

semuelle commented on applications/LootMogul.md:
```diff
@@ -0,0 +1,202 @@
+|    0c. | Testing Guide               | Unit testing will be applied to ensure reliability. Documentation of tests and results will be provided                                                                                                                      |
+|    0d. | Video                      | We will provide a video demonstration that will illustrate all of the functionality delivered with this milestone.                                                                                                                                                                                                  |
+|    0e. | Article                     | We will publish an **article**/workshop/gitbook that explains our application workflow as well as technical details.                                                                                                                                                                                                   |
+|    1. | Core Foundation and Infrastructure Setup                     |  * GitHub/Lab Polkadot dev branch & configure supporting APIs. <br />                                                                    * Finalize requirements with internal game developers & KOL                                                                                                                                   ...
```
What APIs are we talking about here?

semuelle commented on applications/LootMogul.md:
```diff
@@ -0,0 +1,202 @@
+- Polygon Matic
+- Strapi CMS
+- Rust 
+- Solidity
```
```suggestion
- Solidity
- ink!
```

nilesh-sukalikar commented:
> Hi @cruikshankss  => 
is it possible to connect with you team to understand technical details, our business model and platform on which we are working which we also want to bring it to polakadpot ecosystem. 

Would be great if you schdule a call with your team which will also help us to move forward.

cruikshankss commented:
@nilesh-sukalikar Let me talk with my team first about how we can best support you and then I'll get back to you. Hope you have a nice weekend and thanks again for your application. :)

Noc2 reviewed (changes requested):
I still think we would need more technical details, before it makes sense to schedule the call. Most of the above question by me and my colleagues are still not answered (e.g. “Game development and Guild integration”) and they actually need to be defined in the contract and not during a call.

cruikshankss commented:
Dear @nilesh-sukalikar, like @Noc2 said, we expect you to pursue the technical education needed for a successful proposal. Here are some examples of technical education: [Blockchain Academy](https://twitter.com/ParityTech/status/1527313749228457988), [Polkadot wiki](https://wiki.polkadot.network/), [w3f YouTube channel](https://www.youtube.com/c/WEB3Foundation), [Parity Technologies YouTube channel](https://www.youtube.com/c/ParityTech). Best of luck!

RajNextgentm commented:
> Dear @nilesh-sukalikar, like @Noc2 said, we expect you to pursue the technical education needed for a successful proposal. Here are some examples of technical education: [Blockchain Academy](https://twitter.com/ParityTech/status/1527313749228457988), [Polkadot wiki](https://wiki.polkadot.network/), [w3f YouTube channel](https://www.youtube.com/c/WEB3Foundation), [Parity Technologies YouTube channel](https://www.youtube.com/c/ParityTech). Best of luck!

Hi @Noc2 => 
Thanks for sharing these details with us to understand polakadot ecosystem. We will definately go through this.
We recently made some change to our milestone plan, spending more time understanding polakadot ecosystem more into depth. 
I would really appreciate if you review it and provide us the feedback for the same.

cruikshankss commented:
@RajNextgentm Thank you so much for updating with some technical details but I feel I would need a much greater level of technical detail to approve. Very sorry about this.

RajNextgentm commented:
> Thanks again for the update. But I agree with @cruikshankss. The deliverables are basically the requirements of our contract and need to be evaluated, once you submit the milestone. For example: “Writing smart contracts for NFT minting, transfer, approve.” is very unspecified and why are you not using one of the existing ink smart contracts: https://github.com/paritytech/ink/tree/master/examples? And the frontend that you deliver as part of the grant could be basically anything at this stage. Do you have any mockups, etc.?

Thanks @Noc2 for your feedback. 
We are live and we were using third party tool to mint out NFts, now we stopped using that service. All we are trying to do is , brining all our NFT platform to the polakadot ecosystem. 
You can check our application , we are live and minting out NFts on oeensea. So inorder to build our own primary NFT marketplace we want to utilize polkadot's ecossytem.
https://lootmogul.com/nfts

Noc2 reviewed (changes requested):
Thanks for the quick reply. Maybe there is a misunderstanding here. We support the development of open source software and research that is useful for our ecosystem, **doesn’t already exist** and can potentially be used by other projects. We don’t pay projects to join our ecosystem.

semuelle commented:
Still waiting for replies to my questions [here](https://github.com/w3f/Grants-Program/pull/981#pullrequestreview-1024838635), @RajNextgentm.

Noc2 commented:
I’m closing this application due to inactivity. Let me know if I should reopen it.
